salt/proxy/napalm.py

```diff
@@ -194,13 +194,18 @@ def init(opts):
     """
     Opens the connection with the network device.
     """
-    __context__["napalm_device"] = {
-        "opts": opts,
-        "id": opts["id"],
-        "network_device": salt.utils.napalm.get_device(opts),
-        "details": {"initialized": True},
-    }
-    return True
+    try:
+        napalm_device = salt.utils.napalm.get_device(opts)
+        __context__["napalm_device"] = {
+            "opts": opts,
+            "id": opts["id"],
+            "network_device": salt.utils.napalm.get_device(opts),
+            "details": {"initialized": True},
+        }
+        return True
+    except Exception as error:  # pylint: disable=broad-except
+        log.warn("Unable to run init.")
```

Comment:
```suggestion
        log.warn("Unable to run init: %s", error)
```
And remove the above pylint disable.